src/resources/team.yaml

```diff
@@ -52,6 +52,18 @@ actions:
       Returns the compact records for all teams in the organization visible to
       the authorized user.
 
+  - name: findByUser
+    class: get
+    method: GET
+    path: "/users/%s/teams"
+    collection: true
+    params:
+      - name: user
+        <<: *Param.UserId
```

Comment:
User params are special because we can use "me" or an email. Change this to match other user params we have instead of the new UserId.